tailwind.config.cjs

```diff
@@ -8,6 +8,10 @@ module.exports = {
         "highlight-blue": "#0077CC",
         "fade-grey":"#FFFFFFB2",
       },
+      fontFamily: {
+        rajdhani: ['"Rajdhani"', 'sans-serif'],
+        orbitron: ['"Orbitron"', 'sans-serif']
+      }
```

Comment:
Use idiomatic names for the font familites, like "header", "sub-heading", "body" etc.